src/Mapbender/CoreBundle/Resources/public/mapbender.element.legend.js

```diff
@@ -351,7 +361,7 @@
                 if(layers[layidx].children.length > sublayidx) {
                     if(layers[layidx].children[sublayidx].legend) {
                         $(self.element).find("#imgtest").html(
-                                '<img id="testload" style="display: none;" src="' + layers[layidx].children[sublayidx].legend.url + '"></img>');
+                            '<img id="testload" style="display: none;" src="' + layers[layidx].children[sublayidx].legend.url + '"></img>');
```

Comment:
If this would work (but it can't because of the problem with `layidx` etc.), we would create images with the same ID.